common/djangoapps/philu_commands/management/commands/send_referral_follow_up_email.py

```diff
@@ -5,35 +5,40 @@
 from django.utils import timezone
 
 from openedx.features.smart_referral.models import SmartReferral
-from openedx.features.smart_referral.tasks import task_send_referral_follow_up_email
+from openedx.features.smart_referral.tasks import task_send_referral_follow_up_emails
 
 CONTACT_EMAIL = 'contact_email'
 DAYS_TO_SEND_FOLLOW_UP_EMAIL = 3
 
 
 class Command(BaseCommand):
     help = """
-    Send follow up referral email to all those contacts
+    Send follow-up referral email to all those contacts
     """
 
     def handle(self, *args, **options):
         emails_list = []
-        three_days_old_date = timezone.now().date() - timedelta(days=DAYS_TO_SEND_FOLLOW_UP_EMAIL)
+        referral_threshold_date = timezone.now() - timedelta(days=DAYS_TO_SEND_FOLLOW_UP_EMAIL)
 
-        three_days_old_referral = SmartReferral.objects.filter(
-            created__date__lte=three_days_old_date,
+        three_days_old_referral_emails = SmartReferral.objects.filter(
+            created__lte=referral_threshold_date,
             is_referral_step_complete=False
-        ).values(
-            CONTACT_EMAIL
+        ).values_list(
+            CONTACT_EMAIL,
+            flat=True
         )
-        for referral in three_days_old_referral:
+
+        # Removing duplicate email entries.
+        three_days_old_referral_emails = list(set(three_days_old_referral_emails))
```

Comment:
If possible lets use `.distinct()` in the query rather than using set to remove duplicates

Reply:
Upon using `.distinct()`, we can also remove the comment as it would be obvious that we are removing duplicates with the use of distinct.

Reply:
Thanks, done